plugin/trino-hive/src/test/java/io/trino/plugin/hive/util/TestHiveBucketing.java

```diff
@@ -74,6 +74,10 @@
     @Test
     public void testHashingCompare()
     {
+        assertBucketEquals("string", "Trino rocks", 1132136730, -399107423);
+        assertEquals(HiveBucketing.getBucketNumber(1132136730, 4), 2);
+        assertEquals(HiveBucketing.getBucketNumber(-399107423, 4), 1);
```

Comment:
These two are redundant as this is done by `assertBucketEquals` implicitly, isn't it?